tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{36,37},pypy,codechecks
+envlist = py{36,37},pypy3,codechecks
```

Comment:
@mfalesni one more thing If possible to include in same PR l.e. `py35`. As python-3.5 still not retired by python community.

Reply:
Possibly.

Reply:
Actually that is outside of the scope or this PR. This PR just declutters the exceptions and swaps pypy for pypy3 so it only is tested on py3. If they did not update their pypy containers then until recently the latest stable version of pypy3 was the one with compatibility level of Python 3.5.